### PR TITLE
New grayscale palette.

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
PTAL @michaelkirk 

Note that hairlines are now more subtle in "light theme".

### CODE REVIEW

File: SignalMessaging/categories/Theme.m [hunk 1]
```diff
@@ -49,7 +49,7 @@ + (void)setIsDarkThemeEnabled:(BOOL)value
 
 + (UIColor *)backgroundColor
 {
-    return (Theme.isDarkThemeEnabled ? UIColor.ows_blackColor : UIColor.ows_whiteColor);
+    return (Theme.isDarkThemeEnabled ? UIColor.ows_gray95Color : UIColor.ows_whiteColor);
```
Comment (michaelkirk-signal) [inline SignalMessaging/categories/Theme.m]: Is this right?

I see "Dark: **Mobile** Background" should be "#000000"
Whereas gray95 is for "Dark: **Desktop** Background"

File: SignalMessaging/categories/UIColor+OWS.m [hunk 2]
```diff
@@ -142,72 +142,54 @@ + (UIColor *)ows_redColor
...
-    return [UIColor colorWithRGBHex:0x8B8E91];
-}
-
-+ (UIColor *)ows_light60Color
-{
-    return [UIColor colorWithRGBHex:0x62656A];
-}
-
-+ (UIColor *)ows_light90Color
++ (UIColor *)ows_gray15Color
```
Comment (michaelkirk-signal) [inline SignalMessaging/categories/UIColor+OWS.m]: I don't see this one in the spec

File: SignalMessaging/categories/Theme.m (hunk 1 above)
Comment (michaelkirk-signal) [reply SignalMessaging/categories/Theme.m]: This was my only real concern - feel free to merge after addressing.

Comment (charlesmchen-signal) [reply SignalMessaging/categories/Theme.m]: This is pending a final decision from Myles - he's still mulling - but this is supposedly the new design.

File: SignalMessaging/categories/UIColor+OWS.m (hunk 2 above)
Comment (charlesmchen-signal) [reply SignalMessaging/categories/UIColor+OWS.m]: Ah yes, we just eliminated that one after Myles reviewed this work.

**Comment by michaelkirk** [comment]
Ok, as long as it’s Myles approved, LGTM

**Comment by charlesmchen-signal** [comment]
Myles has committed to changing the background color.

Also, he's refined gray 95: https://github.com/signalapp/Signal-iOS/pull/3888/commits/6a712366aecbddf74da6a99778fa1d95803852be

**Pull request merged**